packages/cozy-harvest-lib/src/helpers/windowWrapperMocks.js

```diff
@@ -0,0 +1,30 @@
+import MicroEE from 'microee'
+
+function PopupMock() {
+  this.closed = false
+
+  this.focus = jest.fn()
+  this.close = jest.fn().mockImplementation(() => {
+    this.closed = true
+  })
+
+  this.addEventListener = jest.fn().mockImplementation((name, fn) => {
+    this.on(name, fn)
+  })
+  this.removeEventListener = jest.fn().mockImplementation((name, fn) => {
+    this.removeListener(name, fn)
+  })
+
+  this.clear = () => {
+    this.closed = false
+    this.removeAllListeners()
+    this.focus.mockClear()
+    this.close.mockClear()
+    this.addEventListener.mockClear()
+    this.removeEventListener.mockClear()
+  }
+}
+
+MicroEE.mixin(PopupMock)
+
+export default PopupMock
```

Comment:
nit: is it possible to rewrite the commits in order to have: 
- the revert 
- the move to a functional component
- the add of this OAuthService 

It'll be more easier to read and understand. Also, it'll be easier to maintain later

Reply:
OAuthWindow still exists and is still used in OAuthForm and KonnectorAccountTabs at the moment.

I added the revert, but cannot separate a move to a functional component in a separated commit. I add added a todo in the description for the migration of OAuthForm and KonnectorAccountTabs to OAuthService.

I would like to release a new version of harvest and then a new beta of home and banks and maif for a non regression before migrating and cleaning other components if possible